### PR TITLE
added the html and css files

### PR DESCRIPTION
**Pull request by Tobby8629**
This is just a review of an old project (admission trial project), this project was built with:
- html
- css
- icons from font awesome

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+    <header id="home" style="background-image:linear-gradient(rgba(0,0,50,0.7), rgba(0, 0,50,0.7)), url(image/pic5.jpg);">
+       <div class="nav">
```
Comment (Tobby8629) [inline index.html:31]: Very good navbar but I can be better, see about using the nav tag instead of div with a class of nav

File: index.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+    <header id="home" style="background-image:linear-gradient(rgba(0,0,50,0.7), rgba(0, 0,50,0.7)), url(image/pic5.jpg);">
```
Comment (Tobby8629) [inline index.html:30]: Please avoid the use of inline style 
Keep your style definition in a separate css file

File: index.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+               <a href="#contact"> contact </a>
+           </div>
+       </div>
+       <div class="body">
+           <h2>what is libra book?</h2>
+           <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+           <button>place an order</button>
+       </div>
+   </header>
+   <main id="product">
```
Comment (Tobby8629) [inline index.html:49]: It would be nicer if it was in a section tag instead of main

File: index.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+            <div class="small">
+                <img src="image/pic2.jpeg" alt="">
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+            </div>
+            <div class="small">
+                <img src="image/pic3.jpeg" alt="">
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+            </div>
+        </div>
+
```
Comment (Tobby8629) [inline index.html:82]: This blank lines seem unnecessary, please rectify

File: index.html (hunk 2 above)
Comment (Tobby8629) [inline index.html:30]: Using the class attribute would have been better

File: style.css [hunk 5]
```diff
@@ -0,0 +1,279 @@
+{
```
Comment (Tobby8629) [inline style.css:1]: The blank lines seem unnecessary, please rectify

File: style.css [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+    padding: 0px;
+    margin: 0px;
+    box-sizing: border-box;
+
+}
+body{
+    font-family: 'Open Sans', sans-serif;
+    background-color: azure;
+}
+#home{
```
Comment (Tobby8629) [inline style.css:11]: Like I said class attributes would have been better please rectify this too

File: style.css [hunk 7]
```diff
@@ -0,0 +1,279 @@
...
+    background-attachment: fixed;
+    position: relative;
+}
+.nav{
+    display: flex;
+    justify-content:space-between;
+    align-items: center;
+    width: 89%;
+    margin: auto;
+
```
Comment (Tobby8629) [inline style.css:24]: The Blank space seem unnecessary

File: style.css [hunk 8]
```diff
@@ -0,0 +1,279 @@
...
+}
+.logo{
+    font-size: 30px;
+    color: white;
+    display: flex;
+    margin-top: 25px;
+
```
Comment (Tobby8629) [inline style.css:31]: The blank line seem unnecessary

File: style.css [hunk 9]
```diff
@@ -0,0 +1,279 @@
...
+}
+.logo p{
+    margin-top: 0px;
+
```
Comment (Tobby8629) [inline style.css:35]: The blank line seem unnecessary

File: style.css [hunk 10]
```diff
@@ -0,0 +1,279 @@
...
+}
+.logo i{
+    margin: 5px;
+    color:rgb(40, 164, 196);
+
```
Comment (Tobby8629) [inline style.css:40]: The blank line seem unnecessary

File: style.css [hunk 11]
```diff
@@ -0,0 +1,279 @@
...
+.link a{
+    text-decoration: none;
+    text-transform: capitalize;
+    font-size: 20px;
+    color:aliceblue;
+    margin-right:10px;
+  transition: 0.3s linear}
+
+    .body{
+
```
Comment (Tobby8629) [inline style.css:55]: The blank line seem unnecessary

File: style.css [hunk 12]
```diff
@@ -0,0 +1,279 @@
...
+  margin-top: 0px;
+  padding: 0px;
+}
+.link a{
+    text-decoration: none;
+    text-transform: capitalize;
+    font-size: 20px;
+    color:aliceblue;
+    margin-right:10px;
+  transition: 0.3s linear}
```
Comment (Tobby8629) [inline style.css:52]: The blank line seem unnecessary

File: style.css [hunk 13]
```diff
@@ -0,0 +1,279 @@
...
+        font-size: 18px;
+
+    }
+    .body h2{
+        color:rgb(40, 164, 196);
+        font-size: 40px;
+        margin: 10px 0px;
+    }
+    .body p{
+
```
Comment (Tobby8629) [inline style.css:71]: The blank line seem unnecessary

File: index.html [hunk 14]
```diff
@@ -0,0 +1,95 @@
...
+   <main id="product">
+    <h1>our product</h1>
+    <div class="inner">
+        <div class="pict">
+          <iframe id="video" src="https://www.youtube.com/embed/JEwOy5Fm1yw"></iframe>
+        </div>
+        <div class="text">
+            <h2>quality of our books?</h2>
+            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+
```
Comment (Tobby8629) [inline index.html:58]: The blank line seem unnecessary

File: index.html [hunk 15]
```diff
@@ -0,0 +1,95 @@
...
+        </div>
+    </div>
+
+</main>
+<section id="how-it-work">
+
```
Comment (Tobby8629) [inline index.html:64]: The blank line seem unnecessary

File: style.css [hunk 16]
```diff
@@ -0,0 +1,279 @@
...
+    /*********************content css*****************************/
+#product{
+    width: 90%;
+    margin: 10px auto;
+     padding:0px;
+}
+.inner{
+background-color: rgb(81, 167, 189);
+
```
Comment (Tobby8629) [inline style.css:100]: Please make use of errors to avoid the css error

File: style.css [hunk 17]
```diff
@@ -0,0 +1,279 @@
...
+}
+.divide{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    padding:10px;
+    width: 97%;
+    margin: auto;
+
+
```
Comment (Tobby8629) [inline style.css:152]: Please add linter files to your repo to avoid css errors

**Review by Tobby8629 — COMMENTED** [review]
You did very good with the project and very close to getting approved.
Please add a linter file to your repo to help with linter error and I think it better to use semantic tags

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @Tobby8629 ,

Your are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
 - Good use of inline comments :+1: 
 - Polite review comments :+1: 
 - Good use of Github review feature :+1: 
## Optional suggestions :small_airplane: 
- [ OPTIONAL] Kindly add a good review summary with status in title :factory_worker: 
- [OPTIONAL] It would be better if you used more emojis to make it look more friendly :smile: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**